repository/RediStick-Core.package/RsRedisEndpoint.class/instance/hMSet.dictionary..st

```diff
@@ -2,5 +2,5 @@ commands-hashes
 hMSet: aKey dictionary: aDictionary
 	^ self
 		unifiedCommand:
-			{'HMSET'.
-			aKey} , (self keysAndValuesInlined: aDictionary)
\ No newline at end of file
+			{'HMSET'. aKey} , (self keysAndValuesInlined: aDictionary)
```

Comment:
It would be better to submit this fix as another PR. It is not related to authentication.